bin/test-lxd-images

```diff
@@ -189,6 +189,20 @@ for url in $(lxc query "/1.0/instances" | jq -r .[]); do
         continue
     fi
 
+    # Systemd cleanliness
+    if lxc exec "${name}" -- sh -c "type systemctl" >/dev/null 2>&1; then
+        if lxc exec "${name}" -- systemctl --failed 2>&1 | grep -q '\sfailed\s'; then
+            echo "FAIL: systemd clean: ${name}"
+            # Show the systemd failures
+            lxc exec "${name}" -- systemctl --failed
+            FAIL=1
+        else
+            echo "PASS: systemd clean: ${name}"
+        fi
+    else
+        echo "SKIP: systemd clean: ${name}"
+    fi
```

Comment:
I don't mind the re-order but this won't really change anything in the end as we don't skip the other tests on failure.